splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -78,6 +78,7 @@ public SpliceCatalogUpgradeScripts(SpliceDataDictionary sdd, TransactionControll
         this.tc=tc;
 
         scripts = new ArrayList<>();
+        addUpgradeScript(baseVersion4, 2020, new UpgradeAddConglomerateNumberIndex(sdd, tc));
```

Comment:
Are you sure this should be executed before the `UpgradeConglomerateTable` script? Have you tested a pre-1996 upgrade?

Reply:
Tested with 1990 and the upgrade went smoothly.